src/pybind/mgr/rook/ci/run-rook-e2e-tests.sh

```diff
@@ -6,4 +6,5 @@ set -ex
 : ${CEPH_DEV_FOLDER:=${PWD}}
 ${CEPH_DEV_FOLDER}/src/pybind/mgr/rook/ci/scripts/bootstrap-rook-cluster.sh
 cd ${CEPH_DEV_FOLDER}/src/pybind/mgr/rook/ci/tests
+python3 -m pip install behave
```

Comment:
should we have a requirements file to install deps for tests rather than installing them on the go?